### PR TITLE
fix(interactive): Implement `HttpIrMetaReader` to Get Meta Data From Remote Http Service

### PR DESCRIPTION
**Pull request by shirly121**
## What do these changes do?
1. Implement `HttpIrMetaReader` to get meta data from remote http service.
2. Implement `DynamicIrMetaFetcher` to fetch schema or statistics periodically, the interval can be set by configurations.

## Related issue number

Fixes

### CODE REVIEW

**Review by siyuan0322 — APPROVED** [review]

File: interactive_engine/groot-server/src/main/java/com/alibaba/graphscope/groot/servers/ir/GrootMetaFetcher.java [hunk 1]
```diff
@@ -1,45 +0,0 @@
-/*
```
Comment (longbinlai) [inline interactive_engine/groot-server/src/main/java/com/alibaba/graphscope/groot/servers/ir/GrootMetaFetcher.java:1]: The file should be completely removed?

File: flex/tests/hqps/engine_config_test.yaml [hunk 2]
```diff
@@ -23,6 +23,13 @@ compiler:
       - FilterIntoJoinRule
       - FilterMatchRule
       - NotMatchToAntiJoinRule
+  meta:
+    reader:
+      mode: local
```
Comment (longbinlai) [inline flex/tests/hqps/engine_config_test.yaml:28]: This still runs in local mode?

File: interactive_engine/compiler/conf/ir.compiler.properties [hunk 3]
```diff
@@ -61,3 +61,15 @@ calcite.default.charset: UTF-8
 
 # set the max capacity of the result streaming buffer for each query
 # per.query.stream.buffer.max.capacity: 256
+
+# set the mode to read ir meta, either from a 'local' file or via remote 'http'.
+# ir.meta.reader.mode: local
```
Comment (longbinlai) [inline interactive_engine/compiler/conf/ir.compiler.properties:66]: The configuration of ir.meta.xx, should be uniformly defined with graph.schema and graph.statistics. In addition, we can really configure those service scheme, to automatically differentiate from local and http. For example, 

graph.meta.statistics.uri: http://localhost:8080  -> this is a http file, or 
graph.meta.statistics.uri: file:///path/to/local/file -> this is a local file

Also give a uri for schema

Comment (shirly121) [reply interactive_engine/compiler/conf/ir.compiler.properties]: done

**Review by BingqingLyu — DISMISSED** [review]

**Review by BingqingLyu — APPROVED** [review]

**Review by siyuan0322 — DISMISSED** [review]

**Review by zhanglei1949 — APPROVED** [review]

**Pull request merged**